src/dataflow/utilities/node.ts

```diff
@@ -276,29 +268,9 @@ export const ProgramRunTimes: ProgramRunTime[] = [
     val: 3600
   },
   {
-    text: "2 Hours",
-    val: 7200
-  },
-  {
-    text: "4 Hours",
-    val: 14400
-  },
-  {
-    text: "8 Hours",
-    val: 28800
-  },
-  {
-    text: "24 Hours",
+    text: "1 Day",
     val: 86400
   },
-  {
-    text: "5 days",
-    val: 432000
-  },
-  {
-    text: "10 Days",
-    val: 864000
-  },
   {
     text: "30 Days",
```

Comment:
What would you think of temporarily disabling 1 day and 30 days, until we can automatically set a minimum save-data interval that wont kill our servers? It's currently way too easy to accidentally set off a process that will fill in 2,592,000 rows in the table.

Reply:
Sounds good.  I commented 1 day and 30 days out with a note to add them back in once we've discussed min save data intervals.  No reason to risk flooding the database while folks are testing, tinkering, etc.